vmdb/app/models/storage.rb

```diff
@@ -87,7 +87,7 @@ def active_hosts_with_credentials_in_zone(zone_name)
   end
 
   def active_hosts_with_credentials
-    self.active_hosts.select(&:authentication_valid?)
+    self.active_hosts.select(&:has_credentials?)
```

Comment:
This is called when we want to scan the storage... seems like another place to use authentication_status_ok? instead.